docgen/generate-docs.js

```diff
@@ -271,6 +305,12 @@ function fixAllLinks(htmlFiles) {
  *    links as needed.
  * 5) Check for mismatches between TOC list and generated file list.
  */
+
+var typeMap = new Map();
+typeMap.set('DocumentSnapshot', `https://googleapis.dev/nodejs/firestore/latest/DocumentSnapshot.html`);
+typeMap.set('UserRecord', 'https://firebase.google.com/docs/reference/admin/node/admin.auth.UserRecord.html');
```

Comment:
I think you should only rewrite fully qualified names like `firebase.auth.UserRecord`. Short names like `UserRecord` refers to the type alias defined in the the functions code itself.

Reply:
So if there's a reference to just `DocumentSnapshot`, like [here](https://firebase.devsite.corp.google.com/docs/reference/functions/providers_firestore_.html#change-constructor), should I replace the text with `firebase.firestore.DocumentSnapshot` or just not add a link at all?